RiotSwiftUI/Modules/Room/PollEditForm/View/PollEditForm.swift

```diff
@@ -0,0 +1,153 @@
+                ScrollView {
+                    VStack(alignment: .leading, spacing: 16.0) {
+                        Text(VectorL10n.pollEditFormPollQuestionOrTopic)
+                            .font(theme.fonts.title3)
+                            .foregroundColor(theme.colors.primaryContent)
+                        
+                        VStack(alignment: .leading, spacing: 8.0) {
+                            Text(VectorL10n.pollEditFormQuestionOrTopic)
+                                .font(theme.fonts.subheadline)
+                                .foregroundColor(theme.colors.primaryContent)
+                            
+                            MultilineTextField(VectorL10n.pollEditFormInputPlaceholder, text: $viewModel.question.text)
+                        }
+                        
+                        Text(VectorL10n.pollEditFormCreateOptions)
+                            .font(theme.fonts.title3)
+                            .foregroundColor(theme.colors.primaryContent)
+                        
+                        ForEach(0..<viewModel.answerOptions.count, id: \.self) { index in
+                            SafeBindingCollectionEnumerator($viewModel.answerOptions, index: index) { binding in
+                                AnswerOptionGroup(binding: binding.text, index: index) {
+                                    viewModel.send(viewAction: .deleteAnswerOption(viewModel.answerOptions[index]))
+                                }
+                            }
+                        }
+                        
+                        Button(VectorL10n.pollEditFormAddOption) {
+                            viewModel.send(viewAction: .addAnswerOption)
+                        }
+                        .disabled(!viewModel.viewState.addAnswerOptionButtonEnabled)
+                        
+                        Spacer()
+                        
+                        Button(VectorL10n.pollEditFormCreatePoll) {
+                            viewModel.send(viewAction: .create)
+                        }
+                        .buttonStyle(PrimaryActionButtonStyle(enabled: viewModel.viewState.confirmationButtonEnabled))
+                        .disabled(!viewModel.viewState.confirmationButtonEnabled)
+                    }
+                    .animation(.easeInOut(duration: 0.2))
+                    .padding()
+                    .frame(minHeight: proxy.size.height) // Make the VStack fill the ScrollView's parent
+                    .modifier(NavigationTitleBar(titleColor: theme.colors.primaryContent) {
+                        viewModel.send(viewAction: .cancel)
+                    })
+                }
+            }
+        }
+        .accentColor(theme.colors.accent)
+    }
+}
+
+@available(iOS 14.0, *)
+private struct AnswerOptionGroup: View {
+    
+    @Environment(\.theme) private var theme: ThemeSwiftUI
+    
+    @State private var focused = false
+    
+    var binding: Binding<String>
```

Comment:
Can we use the `@Binding` wrapper here and maybe change the variable name? "text"?